FinniversKit/Sources/Cells/SavedSearchShelfCell/SavedSearchShelfViewModel.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+public struct SavedSearchShelfViewModel {
+    public let title: String
+    public let imageUrlString: String?
+    public let id: Int
+    public let adId: Int
+    public let created: Date
+    
+    public init(id: Int = Int.random(in: 0...100_000), title: String, imageUrlString: String?, adId: Int = Int.random(in: 0...100_000), created: Date = Date()) {
```

Comment:
What is the random int for in `id`? is it for making the view model "unique" in the snapshot? `UUID()` is often used for that purpose 😄 

I don't think we should ever make the `adId` random though, I'd remove this `adId: Int = Int.random(in: 0...100_000)`   and make it random from the caller side instead if it was for demo data

Reply:
wait, when I think about it, why is there an `adId` for saved search? 🤔 
is "id" the saved search id? that might not be unique